packages/search-ui-elasticsearch-connector/src/handlers/search/__tests__/Configuration.test.ts

```diff
@@ -310,6 +310,36 @@ describe("Search - Configuration", () => {
       });
     });
 
+    it("should return the additional headers in the connection options", () => {
+      const state: RequestState = {
+        searchTerm: "test"
+      };
+
+      expect(
+        buildConfiguration({
+          state,
+          queryConfig: { ...queryConfig, facets: null },
+          host,
+          index,
+          apiKey,
+          headers: {
+            Authorization: "Bearer 123",
+            "x-elastic-client-meta": "overridden by build configuration"
+          }
+        })
+      ).toEqual(
+        expect.objectContaining({
+          connectionOptions: {
+            apiKey: "apiKey",
+            headers: {
+              Authorization: "Bearer 123",
+              "x-elastic-client-meta": `ent=${LIB_VERSION}-es-connector,js=browser,t=${LIB_VERSION}-es-connector,ft=universal`
```

Comment:
I intentionally ensured you could not override this header.

Not sure if in practice it actually matters but wrote this test to ensure there is a conscious decision to change the behaviour later on if this method is refactored to flip around the additional headers spread and `x-elastic-client-meta` property in the `buildConfiguration` method.